unstructured/chunking/title.py

```diff
@@ -1,7 +1,14 @@
+"""Implementation of chunking.
```

Comment:
Implementation of chunking by title? (since module name is title.py)

Reply:
Yep, good call, fixed this.